pkg/cmd/audit/audit_filter.go

```diff
@@ -222,13 +222,30 @@ func URIToParts(uri string) (string, schema.GroupVersionResource, string) {
 			return ns, gvr, name
 		}
 
-		if parts[2] != "namespaces" {
+		switch{
```

Comment:
just curious whether client-go or apimachinery already has a function (covered by unit tests) that deconstructs a given url.

Reply:
> just curious whether client-go or apimachinery already has a function (covered by unit tests) that deconstructs a given url.

it's in k8s.io/apiserver I think.

Reply:
what we have now works; as a future goal I am happy to take this as a TODO for me to utilize the function in `k8s.io/apiserver`.